components/user/UserProfile.js

```diff
@@ -68,10 +67,10 @@ function UserProfile({ BASE_URL, data }) {
           </div>
         </div>
       </div>
-      <div className="flex justify-center my-4 text-center">
-        <ReactMarkdown components={{ a: LinkRenderer }}>
-          {data.bio}
-        </ReactMarkdown>
+      <div className="justify-center my-4 text-center whitespace-pre-wrap">
+  <ReactMarkdown  components={{ a: LinkRenderer }}>
+         {data.bio.replace(/\n/gi, '\n &nbsp;')}
+</ReactMarkdown>
```

Comment:
Formatting is not correct here, using the VScode Prettier plugin with format this automatically for you

```suggestion
        <ReactMarkdown components={{ a: LinkRenderer }}>
          {data.bio.replace(/\n/gi, "\n &nbsp;")}
        </ReactMarkdown>
```